config/routes.rb

```diff
@@ -1,5 +1,23 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: 'pages#home'
```

Comment:
"Pages" is actually a controller that comes with devise,  not entirely sure if anything is dependent on it or not. We can leave it in for now, but  just to be aware that it might be redundant later, when we want to route the homepage as our landing page.